cov.py

```diff
@@ -215,6 +228,12 @@ def setup_parser():
                       help="Number of CSV to process per run")
   parser.add_argument("-s", "--steps", type=str, default="all",
                       help="Run steps: csv,cov", dest="steps")
+  parser.add_argument("--core", type=str, default="",
+                      help="Name of target core")
+  parser.add_argument("--isa", type=str, default="",
```

Comment:
Why --isa and --mabi is needed? I believe we are not compiling or simulating ISS for the coverage flow?

Reply:
That's because I specific custom_target, and run.py requres mabi/isa to be set if custom_target is defined.

Reply:
I think we should remove this requirement in the run.py if the steps don't include gcc_compile and iss_sim

Reply:
Agreed, I'm making this change right now.